spec/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe Api::V1::BooksController, type: :controller do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    context 'When fetching all the books' do
+      let!(:books) { create_list(:book, 3) }
+
+      before do
+        get :index
+      end
+
+      it 'responses with all the books' do
+        expected = ActiveModel::Serializer::CollectionSerializer.new(
+          books, each_serializer: EachBookSerializer
+        ).to_json
+        expect(response.body.to_json) =~ JSON.parse(expected)
+      end
+
+      it 'responds with 200 status' do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+  end
+
+  describe 'GET #show' do
+    context 'When fetching a book' do
+      let!(:book) { create(:book) }
+
+      before do
```

Comment:
Nice job, with the last modifications of the other PR(with the not found status), I think we are able to add specs for that condition, what do you think?

For that we can do something like this:
We have 3 "levels" of words, `describe`, `context` and `it`.
The first one is for methods, i.e `describe 'GET #show'`, the second one can be "mixed" or be used inside another. i.e: When you have to test two contexts, you can make something like this:
```
context  'If the book exist' do
   ...specs
end

context 'if the book does not exist' do
   ...specs
end
```